### PR TITLE
CI: undebug CI image

### PR DESCRIPTION
**Pull request by TriplEight**
please don't forget to remove https://github.com/paritytech/polkadot/pull/3807/files#diff-037ea159eb0a7cb0ac23b851e66bee30fb838ee8d0d99fa331a1ba65283d37f7R18-R20

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -38,9 +38,8 @@ variables:                         &default-vars
   CARGO_INCREMENTAL:               0
   DOCKER_OS:                       "debian:stretch"
   ARCH:                            "x86_64"
-  # FIXME: revert me
-  # CI_IMAGE:                        "paritytech/ci-linux:production"
-  CI_IMAGE:                        "paritytech/ci-linux:staging-1.54.0"
+  FIXME: revert me
```
Comment (ordian) [inline .gitlab-ci.yml:41]:
```suggestion
```

**Review by ordian — APPROVED** [review]

**Comment by ordian** [comment]
bot merge

**Comment by ordian** [comment]
bot merge

**Comment by ordian** [comment]
bot merge

**Review by alvicsam — APPROVED** [review]

**Pull request merged**